packages/server-core/src/seeder.ts

```diff
@@ -55,7 +57,7 @@ export async function seeder(app: Application, forceRefresh: boolean, prepareDb:
       if (fs.existsSync(uploadPath)) fs.rmSync(uploadPath, { recursive: true })
     }
     copyDefaultProject()
-    await app.service(projectPath)._seedProject('default-project')
+    if (config.kubernetes.enabled || config.testEnabled) await app.service(projectPath)._seedProject('default-project')
```

Comment:
we should also reseed on dev-reinit

Reply:
this will happen in `_fetchDevLocalProjects`